s3_scrubber/src/lib.rs

```diff
@@ -204,26 +204,16 @@ pub struct BucketConfig {
     pub region: String,
     pub bucket: String,
     pub prefix_in_bucket: Option<String>,
-
-    /// Use SSO if this is set, else rely on AWS_* environment vars
-    pub sso_account_id: Option<String>,
 }
 
 impl Display for BucketConfig {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(
-            f,
-            "{}/{}/{}",
-            self.sso_account_id.as_deref().unwrap_or("<none>"),
-            self.region,
-            self.bucket
-        )
+        write!(f, "{}/{}", self.region, self.bucket)
     }
 }
```

Comment:
This has to be the weirdest Display ever. A `derive(Debug)` would be much more helpful.

Luckily it is unused, so lets get rid of it. Aargh, cannot suggest.

Reply:
removed.